### DISCUSSION TITLE
pyinstaller executable on other raspberry can't execute

### DISCUSSION THREAD

**Discussion by arponcedeleon** [pyinstaller/pyinstaller#5643] [PyInstaller]
I am having problem when try to execute the executable file generated by pyinstaller in other raspberry device.
Python code: test.py
PyInstaller: pyinstaller test.py --onefile

I have 2 raspberry pi:

#pi-1 I generated the executable with success and here the executable works fine ./dist/test

#pi-2 I did the copy of full /dist folder and go to the terminal and ./dist/test doesn't work:

I got this result ./dist/test: command not found

note: boot PI have the same PI OS and hardware model

How to run the executable generated in other devices??

**Comment by bwoodsend** [comment]
What does `stat dist/TestODBC` say on PI 2?

**Reply by arponcedeleon** [reply]
ohh it is just ./dist/test from file test.py
